macros/SpecName.ejs

```diff
@@ -467,10 +467,6 @@ var specList = {
         name : 'Document Object Model (DOM) Level 3 XPath Specification',
         url  : 'https://www.w3.org/TR/DOM-Level-3-XPath/'
     },
-    'DOM4': {
```

Comment:
Thanks. Since DOM4 is just an outdated fork of WHATWG DOM, instead of e.g. a historical reference like DOM 1/2/3 are, I will remove references to DOM4 from those pages. However I'll leave it in the database marked as obsolete, as DOMError is an obsolete object which does not appear in modern DOM specs, so it is needed for there.